web/components/feed/feed-comments.tsx

```diff
@@ -465,81 +464,43 @@ export function CommentInput(props: {
 
 export function CommentInputTextArea(props: {
   user: User | undefined | null
-  isReply: boolean
   replyToUsername: string
-  commentText: string
-  setComment: (text: string) => void
+  editor: Editor | null
+  upload: any
   submitComment: (id?: string) => void
   isSubmitting: boolean
-  setRef?: (ref: HTMLTextAreaElement) => void
   presetId?: string
-  enterToSubmitOnDesktop?: boolean
 }) {
   const {
-    isReply,
-    setRef,
     user,
-    commentText,
-    setComment,
+    editor,
+    upload,
     submitComment,
     presetId,
     isSubmitting,
     replyToUsername,
-    enterToSubmitOnDesktop,
   } = props
   const { width } = useWindowSize()
-  const memoizedSetComment = useEvent(setComment)
   useEffect(() => {
-    if (!replyToUsername || !user || replyToUsername === user.username) return
-    const replacement = `@${replyToUsername} `
-    memoizedSetComment(replacement + commentText.replace(replacement, ''))
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [user, replyToUsername, memoizedSetComment])
+    editor?.setEditable(!isSubmitting)
+  }, [isSubmitting, editor])
+
+  // TODO: make at mention show up at beginning
   return (
     <>
       <Row className="gap-1.5 text-gray-700">
-        <Textarea
-          ref={setRef}
-          value={commentText}
-          onChange={(e) => setComment(e.target.value)}
-          className={clsx('textarea textarea-bordered w-full resize-none')}
-          // Make room for floating submit button.
-          style={{ paddingRight: 48 }}
```

Comment:
Does the new editor look ok with the mobile submit button?

Reply:
it does